bitrise.yml

```diff
@@ -2774,13 +2774,21 @@ workflows:
 
             brew install jq
 
-            curl -X GET -s -H 'Accept: application/vnd.github.v3+json' -H "authorization: Bearer ${GITHUB_ACCESS_TOKEN}" https://api.github.com/repos/mozilla-mobile/firefox-ios/commits\?sha\=main\&since\=$YESTERDAY | jq -e -r '.[].commit.author | select(.name | contains("BITRISE.YML-bot"))' > /dev/null/
-            test $? -eq 0 || exit 1
+            resp=$(curl -X GET -s -H 'Accept: application/vnd.github.v3+json' -H "authorization: Bearer ${GITHUB_ACCESS_TOKEN}" https://api.github.com/repos/mozilla-mobile/firefox-ios/commits\?sha\=main\&since\=$YESTERDAY | jq -r '.[].commit.author | select(.name | contains("BITRISE.YML-bot"))')
+            echo $resp
+            if [ -z $resp ]
+            then
+                echo "There is not any new commit, stop building"
+            else
+                echo "There is a new commit, continue building"
+                envman add --key NEW_XCODE_VERSION --value New_Version_Found
```

Comment:
Does the value need quotations? Not familiar with ```envman``` key value pairs

Reply:
I started using quotes but then I had to remove them to have the variable working for the run_if conditions in next steps, please see this post: https://discuss.bitrise.io/t/how-can-i-use-variables-stored-with-envman-in-the-run-if-check-before-a-step-is-run/3857/3